theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsSizeSemanticTokens.kt

```diff
@@ -1,101 +1,132 @@
-    val iconWithBodyMediumSizeTallMobile: Float = DimensionRawTokens.dimension300,
-    val iconWithBodyMediumSizeTallTablet: Float = DimensionRawTokens.dimension300,
-    val iconWithBodySmallSizeMediumMobile: Float = DimensionRawTokens.dimension200,
-    val iconWithBodySmallSizeMediumTablet: Float = DimensionRawTokens.dimension200,
-    val iconWithBodySmallSizeShortMobile: Float = DimensionRawTokens.dimension150,
-    val iconWithBodySmallSizeShortTablet: Float = DimensionRawTokens.dimension150,
-    val iconWithBodySmallSizeTallMobile: Float = DimensionRawTokens.dimension250,
-    val iconWithBodySmallSizeTallTablet: Float = DimensionRawTokens.dimension250,
-    val maxWidthTypeDisplayLargeMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeDisplayLargeTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeDisplayMediumMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeDisplayMediumTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeDisplaySmallMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeDisplaySmallTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingLargeMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingLargeTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingMediumMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingMediumTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingSmallMobile: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeHeadingSmallTablet: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeHeadingXlargeMobile: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeHeadingXlargeTablet: Float = DimensionRawTokens.dimension9000,
-    val maxWidthTypeBodyLargeMobile: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeBodyLargeTablet: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeBodyMediumMobile: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeBodyMediumTablet: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeBodySmallMobile: Float = DimensionRawTokens.dimension6000,
-    val maxWidthTypeBodySmallTablet: Float = DimensionRawTokens.dimension6000,
-)
\ No newline at end of file
+  val iconDecorativeMedium: Float = DimensionRawTokens.dimension500,
+  val iconDecorativeShort: Float = DimensionRawTokens.dimension400,
+  val iconDecorativeShorter: Float = DimensionRawTokens.dimension300,
+  val iconDecorativeShortest: Float = DimensionRawTokens.dimension200,
+  val iconDecorativeTall: Float = DimensionRawTokens.dimension600,
+  val iconDecorativeTaller: Float = DimensionRawTokens.dimension700,
+  val iconDecorativeTallest: Float = DimensionRawTokens.dimension900,
+  val iconWithBodyLargeSizeMediumDesktop: Float = DimensionRawTokens.dimension500,
```

Comment:
Just for confirmation @paulinea, do I need to remove the "Desktop" from the files' generation?

Reply:
Yes, we don't use "Desktop" tokens on our side, you can remove them.

Reply:
@paulinea PR closed. Please remove the `tokenator-update-tokens-20241025073011` branch 🙏